core/platform/bulk_email/mailchimp_bulk_email_services.py

```diff
@@ -97,13 +97,12 @@ def _create_user_in_mailchimp_db(
         Exception. Any error (other than the one mentioned below) raised by the
             mailchimp API.
     """
-    post_data = {
-        'email_address': user_email,
-        'status': 'subscribed'
-    }
-
     try:
-        client.lists.members.create(feconf.MAILCHIMP_AUDIENCE_ID, post_data)
+        # Here we use MyPy ignore because the following is a function in the
```

Comment:
This seems not quite OK -- could you please check with @sahiljoster32 what to do here?

Reply:
Hi @seanlip, thanks for the ping!

The argument types that create method can accept are `str` ( for the first argument ) and `Dict[str, str]` ( for the second argument ). Reference where this info is written: https://github.com/VingtCinq/python-mailchimp/blob/5989d226d05a7d53dc76a6558e9c8b0190818f43/mailchimp3/entities/listmembers.py#L44

But if we look at the code carefully ( in the reference link ) we can see that it can accept any Dict type, because `status` and `email_address` are mandatory and it should be string, other than these keys we can provide any type of key-value pair.

So, I think it's ok to change the stub's type according to our need.

Reply:
Done, thanks for the info @sahiljoster32!